src/cs/usfca/edu/histfavcheckout/controller/HistFavCheckoutController.java

```diff
@@ -5,7 +5,6 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.CrossOrigin;
```

Comment:
Looks like your changes are removing these and a bunch of other things that are already on master & shouldn't be removed without good reason.